### ISSUE TITLE
Would be great to have a "Best of ..." combination set in the Accessibility Testcase Profile dropdown

### ISSUE THREAD

**Issue by DavidMacDonald** [wet-boew/wet-boew-wpss#111]
It would be a combined check that has all the WPSS tests and all the AXE rules, but doesn't have any overlap. So where there is overlap, one of the rules would be commented out. I might be able to help with that.

I don't think there is a way to comment out Pa11y rules so in this check I guess pa11y wouldn't be included in this selection from the dropdown.

**Comment by irwink** [comment]
There is an "All accessibility" selection for the "Testcase Profile Groups" or an "All" selection for the "Accessibility Testcase Profile".  Both of these select a testcase profile that is a combination of WCAG 2.0 and Deque AXE.  When checks are performed, if a particular fault is both a WCAG 2.0 and Deque AXE error, only the WCAG 2.0 error message is generated.   The Pa11y checks are not included in this particular profile as there is no easy way to prevent double reporting of errors.

**Comment by DavidMacDonald** [comment]
perfect... I"ll check it out.

**Comment by DavidMacDonald** [comment]
I tried the "All" profile from Accessibility test case profile. Are you sure pa11y is suppressed?

![image](https://user-images.githubusercontent.com/2365411/66675509-74e28400-ec33-11e9-910a-f2a44621adf1.png)

![image](https://user-images.githubusercontent.com/2365411/66675527-7b70fb80-ec33-11e9-8ce3-a1e4a6388438.png)

![image](https://user-images.githubusercontent.com/2365411/66675602-a1969b80-ec33-11e9-97d3-944bf14bfe1f.png)

**Comment by irwink** [comment]
Opps,  Pa11y is included in the All accessibility profile.
